Libraries/src/Amazon.Lambda.RuntimeSupport/Helpers/FileDescriptorLogStream.cs

```diff
@@ -32,8 +32,9 @@ public static class FileDescriptorLogFactory
 
         // max cloudwatch log event size, 256k - 26 bytes of overhead.
         internal const int MaxCloudWatchLogEventSize = 256 * 1024 - 26;
-        internal const int LambdaTelemetryLogHeaderLength = 8;
-        internal const uint LambdaTelemetryLogHeaderFrameType = 0xa55a0001;
+        internal const int LambdaTelemetryLogHeaderLength = 16;
+        internal const uint LambdaTelemetryLogHeaderFrameType = 0xa55a0003;
+        internal const DateTimeOffset UnixEpoch = new DateTimeOffset(1970, 1, 1, 0, 0, 0, TimeSpan.Zero);
```

Comment:
This failed to compile because you can't use `const` when newing a new `DateTimeOffset`. Replace `const` here with `static readonly`.